website/content/docs/connect/config-entries/mesh.mdx

```diff
@@ -47,24 +48,102 @@ TransparentProxy {
 ```
 
 </Tab>
-</Tabs>
-
-## Available Fields
-
-- `Kind` - Must be set to `mesh`
+<Tab heading="Kubernetes YAML">
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: Mesh
+metadata:
+  name: mesh
+spec:
+  transparentProxy:
+    meshDestinationsOnly: true
+```
 
-- `Namespace` `(string: "default")` <EnterpriseAlert inline /> - Specifies the namespace the config entry will apply to.
-  Must be set to `default`
+</Tab>
+<Tab heading="Kubernetes YAML (Consul Enterprise)">
+
+**NOTE:** A `Mesh` resource can be created in any Kubernetes
+namespace but it will apply to proxies across **all** namespaces. Only one
+`Mesh` resource can exist in the cluster.
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: Mesh
+metadata:
+  name: mesh
+spec:
+  transparentProxy:
+    meshDestinationsOnly: true
+```
 
-- `Meta` `(map<string|string>: nil)` - Specifies arbitrary KV metadata pairs.
+</Tab>
+</Tabs>
 
-- `TransparentProxy` `(TransparentProxyConfig: <optional>)` - Controls configuration specific to proxies in
-  `transparent` [mode](/docs/connect/config-entries/service-defaults#mode). Added in v1.10.0.
+## Available Fields
 
-  - `MeshDestinationsOnly` `(bool: false)` - Determines whether sidecar proxies operating in transparent mode can
-    proxy traffic to IP addresses not registered in Consul's mesh. If enabled, traffic will only be proxied
-    to upstream proxies or Connect-native services. If disabled, requests will be proxied as-is to the
-    original destination IP address. Consul will not encrypt the connection.
+<ConfigEntryReference
```

Comment:
Previous docs weren't using the `ConfigEntryReference` react component that renders both kube and hcl docs.